cloudmapper.py

```diff
@@ -49,10 +49,27 @@ def get_account(account_name, config, config_filename):
 def run_gathering(arguments):
     from cloudmapper.gatherer import gather
     parser = argparse.ArgumentParser()
+    parser.add_argument("--config", help="Config file name",
```

Comment:
Are you using the config file for anything?  You could use the config file to identify the default account, but I didn't see you getting that in the code.

Reply:
yes, it is being used below to get the default account name (https://github.com/alanquillin/cloudmapper/blob/904c08861735062e91518aba644f64b8d83c291d/cloudmapper.py#L65)